mne_bids/meg_bids.py

```diff
@@ -343,7 +424,7 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
 
     make_dataset_description(output_path, name=" ",
                              verbose=verbose)
-    _channel_json(raw, task, manufacturer, data_meta_fname, kind, verbose)
+    _sidecar_json(raw, task, manufacturer, data_meta_fname, kind, verbose)
```

Comment:
Renaming `_channel_json` to `_sidecar_json` is actually more than just docstrings - but what you create with this function *is* the kind-specific sidecar json file so I suggest we change it.

Reply:
okay for me